packages/plugins/minos-router-graphql/minos/plugins/graphql/handlers.py

```diff
@@ -60,6 +66,10 @@ def _build_response_from_graphql(result: ExecutionResult) -> Response:
             for error in errors:
                 if isinstance(error.original_error, ResponseException):
                     status = error.original_error.status
+                    error_trace = "".join(traceback.format_tb(error.original_error.__traceback__))
+                    logger.exception(f"Raised a system exception:\n {error_trace}")
+                else:
+                    logger.error(f"Raised an application exception:\n {error}")
```

Comment:
Hi @vladyslav-fenchak! I think you reversed the log messages. In any case, I would use the error_trace in both cases 🙂

Reply:
Done!